internal/internal_worker.go

```diff
@@ -141,6 +141,10 @@ type (
 		DisableStickyExecution bool
 
 		StickyScheduleToStartTimeout time.Duration
+
+		// MismatchedWorkflowPolicy is used for configuring how client's decision task handler deals with
```

Comment:
is used for configuring how decision worker deals with non-deterministic history events (presumably arising from non-deterministic workflow definitions or non-backward compatible workflow definition changes).